### PR TITLE
Stripe form

### PR DESCRIPTION
**Pull request by stefanosu**
- Added styling to PaymentForm, PaymentCancel and PaymentSuccess Components 0_o 
- Fixed bug on frontend payment form 
- Deleted console logs  
- Still need to add unit tests to components

### CODE REVIEW

File: bloggies_frontend/src/RoutedPages/PaymentPage/index.tsx [hunk 1]
```diff
@@ -2,38 +2,28 @@ import React, { ReactEventHandler, useState } from "react";
...
-  const [billingDetails, setBillingDetails] = useState({
-    phone: "",
-    name: ""
-  });
-
 
   const handleSubmit = async (event:any) => {
+    console.log('for the love of GOD');
     event.preventDefault();
+    console.log('for');
```
Comment (dogpineapple) [inline bloggies_frontend/src/RoutedPages/PaymentPage/index.tsx:27]: you can remove the console.logs now!

File: bloggies_frontend/src/RoutedPages/RegisterCustomer/index.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React, { ReactEventHandler, useState } from "react";
+import styled from 'styled-components' 
+import {CreateTokenCardData} from '@stripe/stripe-js';
+import { BASE_URL } from "../../config";
+import { useSelector, useDispatch } from "react-redux";
+import { CustomReduxState } from "../../custom";
+import { createCustomer } from "../../redux/stripeAction";
+
+
+const RegisterCustomer = ()  => {
```
Comment (dogpineapple) [inline bloggies_frontend/src/RoutedPages/RegisterCustomer/index.tsx]: I think we can delete the RegisterCustomer component

Comment (dogpineapple) [reply bloggies_frontend/src/RoutedPages/RegisterCustomer/index.tsx]: (and the route too!)

**Review by dogpineapple — COMMENTED** [review]
Left two comments!

**Review by abquansah — APPROVED** [review]

**Review by megmcdevitt — APPROVED** [review]
Looks good!

**Pull request merged**